readme.md

```diff
@@ -13,7 +13,7 @@ Protractor-sync builds on protractor and provides:
 
 Pre-reqs:
 
-* Protractor (or something like grunt-protractor-runner, which includes it)
+* [Protractor](http://www.protractortest.org/#/) (follow the setup instructions)
```

Comment:
I think it would still be correct to get protractor from another source such as grunt-protractor-runner, so I'm not sure that we should remove that part.

I'm also thinking we probably shouldn't be directing people to follow the Protractor setup docs because I don't think they will work out of the box. To update webdriver for protractor-sync, you need to run `grunt shell:webdriverUpdate`. Running `webdriver-manager update`, as the Protractor docs direct, won't work in the root directory of protractor-sync.